python/lsst/pipe/supertask/cmdLineActivator.py

```diff
@@ -0,0 +1,554 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""
+Module defining CmdLineActivator class and related methods.
+"""
+
+from __future__ import print_function
+from builtins import object
+
+#--------------------------------
+#  Imports of standard modules --
+#--------------------------------
+import sys
+import traceback
+
+#-----------------------------
+# Imports for other modules --
+#-----------------------------
+import lsst.log as lsstLog
+import lsst.pipe.base.argumentParser as pipeBaseArgParser
+from lsst.pipe.base.task import TaskError
+from .activator import profile
+from .parser import makeParser
+from .taskLoader import (TaskLoader, KIND_CMDLINETASK, KIND_SUPERTASK)
+
+
+#----------------------------------
+# Local non-exported definitions --
+#----------------------------------
+
+__all__ = ['CmdLineActivator']
+
+# logging properties
+_logProp = """\
+log4j.rootLogger=INFO, A1
+log4j.appender.A1=ConsoleAppender
+log4j.appender.A1.Target=System.err
+log4j.appender.A1.layout=PatternLayout
+log4j.appender.A1.layout.ConversionPattern={}
+"""
+
+
+def _table(tuples, headers):
+    """Nice formatting of 2-column table
+    """
+    if not tuples:
+        return
+    width = max(len(x[0]) for x in tuples)
+    if headers:
+        width = max(width, len(headers[0]))
+        print(headers[0].ljust(width), headers[1])
+        print(''.ljust(width, '-'), ''.ljust(len(headers[1]), '-'))
+    for col1, col2 in tuples:
+        print(col1.ljust(width), col2)
+
+#------------------------
+# Exported definitions --
+#------------------------
+
```

Comment:
I was actually wrong about LSST conventions, it looks like ordering in this module does agree with that:
https://developer.lsst.io/coding/python_style_guide.html#standard-code-order-should-be-followed